cloud/disk_manager/internal/pkg/persistence/s3_metrics.go

```diff
@@ -51,6 +53,15 @@ func (m *s3Metrics) StatCall(
 		}
```

Comment:
let's add bucket and key to "S3 call hanging"/"S3 call error" logs, it'll help to identify hanging object